portfolio-edit/Panel.php

```diff
@@ -10,67 +10,43 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+<link rel="stylesheet" href="panel.css">
 <body>
 
 <h1> Admin page </h1>
+
+<body>
+<?php
+echo '
+<div class="addNewProject"> <a href="panel.html"> add new project</a></div>';
```

Comment:
why is this echo'd? you can just remove the php here and have normal html

Reply:
ok